client/post-editor/editor-seo-accordion/index.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import CountedTextarea from 'components/forms/counted-textarea';
+import Gridicon from 'components/gridicon';
+import InfoPopover from 'components/info-popover';
+import TokenField from 'components/token-field';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+
+class EditorSeoAccordion extends Component {
+	render() {
+		const { translate } = this.props;
+		// Temporary placeholder chips for design review
+		const sampleChips = [ 'Post Title', 'Site Title' ];
+
+		return (
+			<Accordion
+				title={ translate( 'Advanced SEO' ) }
+				icon={ <Gridicon icon="search" /> }
+				className="editor-seo-accordion"
+			>
+				<AccordionSection>
+					<span className="editor-drawer__label-text">
```

Comment:
Side-note for future maintenance: We shouldn't expect drawer components to add this class tag. Rather, it probably makes sense to create a `<EditorDrawerLabelText />` component or something equivalent.